src/ipaperftest/core/plugin.py

```diff
@@ -140,7 +142,7 @@ def clone_ansible_freeipa(self, ctx):
                 "clone",
                 "--depth=1",
                 "--branch",
-                "v0.3.8",
+                "v1.8.4",
```

Comment:
I'm curious about the huge version jump

Reply:
It's not really as huge as the naming implies. ansible-freeipa jumped directly from 0.4 to 1.5 because of Ansible Automation Hub requirements. I updated to the latest release, since the one we were using was pretty old already.